hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/ethereum/EthereumSuite.java

```diff
@@ -296,6 +295,66 @@ List<HapiSpec> feePaymentMatrix() {
                 .toList();
     }
 
+    @HapiTest
+    HapiSpec matrixedPayerRelayerTest1() {
```

Comment:
I did not know how to add the `@HapiTest` annotation to the tests from `feePaymentMatrix` so I created separate functions. I am open to suggestions on how to do this in a more better way.

Reply:
This looks good to me!